INN/umbrella-borderzine
# Updates to author images displaying

## Changes

This pull request makes the following changes:

- Created new `borderzine_has_avatar` function, copied from https://github.com/INN/largo/blob/f50df16e9e06fb9402802e2f1ce76ce058d26d76/inc/avatars.php#L34-L54 , to include a check to see if `userphoto_the_author_thumbnail()` exists from the [User Photo](https://wordpress.org/plugins/user-photo/) plugin.
- Updates `borderzine_byline` class to use new `borderzine_has_avatar` function.
- Updates `content-single.php` to use `borderzine_byline` class.
- Updates `author-bio-description.php` partial to use new `borderzine_has_avatar` class.

## Why



For #21 / author images not displaying now that we've figured out how to get them on the staging site.

## Testing/Questions

Features that this PR affects:

- Author images
- Author bylines


Questions that need to be answered before merging:

- [ ] Are there any more cases of `largo_has_avatar` we should swap to use `borderzine_has_avatar`?

Steps to test this PR:

1. Install the [User Photo](https://wordpress.org/plugins/user-photo/) plugin and verify authors now have images in their profile
2. View an author byline on a post and verify their picture shows
3. View an author bio widget and verify their picture shows
